src/sources/nats.rs

```diff
@@ -45,13 +45,22 @@ pub struct NatsSourceConfig {
     /// The NATS URL to connect to.
     ///
     /// The URL must take the form of `nats://server:port`.
```

Comment:
Somewhat nitty, but _technically_ the URL must take the form of `nats://server` with an optional `:port`, but I'm not sure how much we really care there.

Reply:
I'll remove the word "must" to make it less definitive.